api/go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	go.opentelemetry.io/otel/trace v1.11.2
 	go.opentelemetry.io/proto/otlp v0.19.0
 	golang.org/x/exp v0.0.0-20221126150942-6ab00d035af9
-	golang.org/x/net v0.2.0
+	golang.org/x/net v0.3.1-0.20221206200815-1e63c2f08a10
```

Comment:
I think we are missing a few direct deps here, sorry for not being more specific:

* go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp
* google.golang.org/genproto